### DISCUSSION TITLE
Is master stable?

### DISCUSSION THREAD

**Discussion by renyhp** [searx/searx#3093] [General]
In the past I have upgraded my searx instance when a release happened, but I see no release since quite some time, and no milestone for a new release. I was wondering if I should really wait for a new release, or master is considered stable to upgrade. Thanks!

**Comment by unixfox** [comment]
Master is somewhat stable, we try to avoid publishing unstable code.

**Reply by renyhp** [reply]
That's good! Is there any kind of guideline on how often to upgrade, or any place to look at in order to be periodically informed on the changelog (other than the commit list, of course)?
